include/prepared_statement.h

```diff
@@ -0,0 +1,46 @@
+#include "uthash.h"
+
+#define MAX_SERVER_PREPARED_STMT_NAME 24
+
+typedef int queryid_t;
```

Comment:
`_t` suffix is reserved by standards. I think we can just use `int`.

Reply:
Used `uint64_t` instead. Because the exact size is important for MAX_SERVER_PREPARED_STMT_NAME (which now has a comment).